test/apps/iodemo/io_demo.cc

```diff
@@ -427,16 +453,19 @@ class DemoClient : public P2pDemoCommon {
         }
 
     private:
-        long                                _counter;
+        long                                _comp_counter;
         long*                               _io_counter;
+        long*                               _conn_io_counter;
+        CounterSet                          _counters;
```

Comment:
1 local counter for this object + CounterSet reference for connection and server